build/AzurePipelineTemplates/CsWinRT-Pipeline.yml

```diff
@@ -0,0 +1,11 @@
+# This file is used as basis for the ADO pipeline
+name: $(MajorVersion).$(MinorVersion).$(PatchVersion)$(PrereleaseVersion).$(date:yyMMdd)$(rev:.r)
+
+stages:
+- template: CsWinRT-BuildAndTest-Stage.yml
+
+- template: CsWinRT-PublishToNuget-Stage.yml
+
+# if we are doing a release build, publish the build to Maestro
+# - ${{ if eq( '$(PrereleaseVersion)', '') }}:

```

Comment:
Does these needs to be uncommented?

Reply:
Yes this is just for testing and will be removed before merge